### DISCUSSION TITLE
SideDiffusiveFluxIntegral

### DISCUSSION THREAD

**Discussion by styyokuda** [idaholab/moose#27222] [Q&A General] [answered]
All,
I am trying to calculate a value of mole [mol] on a boundary from a flux [mol/m^2/sec] using SideDiffusiveFluxIntegral.
It seems that SideDiffusiveFluxIntegral calculates a concentration rate [mol/sec] by integrating a flux [mol/m^2/sec] with respect to an area [m^2].
If so, the time step size needs to be multiplied to the value [mol/sec] calculated from SideDiffusiveFluxIntegral to obtain a value of mole [mol]?

Is there a MOOSE app which integrates a flux with respect with time [sec] and area [m^2]?
Please help.  S. Thomas

**Comment by GiudGiud** [comment, marked as answer]
Hello

You can feed this postprocessor into this one to perform the time integration
https://mooseframework.inl.gov/source/postprocessors/TimeIntegratedPostprocessor.html

Guillaume

**Reply by styyokuda** [reply]
Thank you, Guillaume!
